src/EditorFeatures/Core/Extensibility/NavigationBar/AbstractEditorNavigationBarItemService.cs

```diff
@@ -49,7 +49,7 @@ public Task<bool> TryNavigateToItemAsync(Document document, NavigationBarItem it
             NavigateToPosition(workspace, documentId, position, virtualSpace, cancellationToken);
         }
 
-        protected void NavigateToPosition(Workspace workspace, DocumentId? documentId, int position, int virtualSpace, CancellationToken cancellationToken)
+        protected void NavigateToPosition(Workspace workspace, DocumentId documentId, int position, int virtualSpace, CancellationToken cancellationToken)
```

Comment:
As best as I could tell, this never would actually be passed a null, and it wasn't relying on anything downstream being able to handle it, so I updated this annotation.